src/gui/gui_map_icons.hpp

```diff
@@ -885,6 +923,20 @@ class map_province_values : public window_element_base {
 			return make_element_by_type<map_pv_rail>(state, id);
 		} else if(name == "fort_icon") {
 			return make_element_by_type<map_pv_fort>(state, id);
+		} else if(name == "bank_icon") {
+			if(state.economy_definitions.building_definitions[int32_t(economy::province_building_type::bank)].defined) {
+				return make_element_by_type<map_pv_bank>(state, id);
+			}
+			auto ptr = make_element_by_type<image_element_base>(state, id);
```

Comment:
instead of making an element, making it invisible, and then returning it, you can just make an instance of `invisible_element` and return it immediately